### ISSUE TITLE
Tutorial App Crashes. 3 bugs

### ISSUE THREAD

**Issue by johnMedlockDev** [dotnet/docs-maui#829]
Hi there, I was playing around with the app I build through the tutorial. 

Very neat IMO. 

I noticed that when you:
* Delete the storage of the app, it becomes unrecoverable. It's not like a user wouldn't ever do that. Could you maybe add a solution in the tut for that?
* When you open Chrome through the link, and push the back button it just crashes the app... (Android)

MAUI Bug:
* If you create a project, then close the solution, and go to delete the project. Windows won't allow it because it says the project is still in use. VS 2022 is closed. I know it won't happen often, but having to reboot just to delete the project seems crazy to me.

Love the project!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 720bf7f0-db92-f738-b864-ee149a02377b
* Version Independent ID: 720bf7f0-db92-f738-b864-ee149a02377b
* Content: [Tutorial: Create a .NET MAUI app - .NET MAUI](https://docs.microsoft.com/en-us/dotnet/maui/tutorials/notes-app/)
* Content Source: [docs/tutorials/notes-app/index.yml](https://github.com/dotnet/docs-maui/blob/main/docs/tutorials/notes-app/index.yml)
* Product: **dotnet-mobile**
* Technology: **dotnet-maui**
* GitHub Login: @adegeo
* Microsoft Alias: **adegeo**

**Comment by adegeo** [comment]
Hi! Thank you for the feedback. I'm assuming that you mean [Tutorial: Create a .NET MAUI app - .NET MAUI](https://docs.microsoft.com/en-us/dotnet/maui/tutorials/notes-app/) ?

For the Visual Studio bug, you should use the **Help** > **Send Feedback** > **Report a problem...**

**Comment by johnMedlockDev** [comment]
Hi Andy, yes that's it.

**Comment by adegeo** [comment]
Ok thanks. I'm collecting feedback on the tutorial series and will address them soon. Thanks again!

**Comment by adegeo** [comment]
Hi @johnMedlockDev I checked on the BACK BUTTON crashing and it happens in a non .NET MAUI app when navigating back. For example, I opened up the text message app, sent a message to a random number which fails to send. The message is in the sent list and contains a link. I click on the link which opens Chrome and then I navigate back to the message app and the emulator crashes. I used the feedback mechanism to send a crash report. Nothing we can do here though. I'm going to close this issue. Thanks again for the feedback.
